### PR TITLE
Create releasing-patch-update.md

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

**Review by RedGuy12 — APPROVED** [review]
(sorry 😳 )

File: content/docs/develop/releasing-patch-update.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+16. On your local environment, on `v1.xx.x` branch, manually bring the translations and commit.
+17. On your local environment, on `v1.xx.x` branch, bump to v1.xx.1 and commit.
+18. Run `git push origin v1.xx.x`.
+19. **The v1.xx.x branch is now ready for use.**
+
+## Why hyphen-master branch?
+This branch is used to prevent merge conflict fixes from accidentally leaking into v1.xx.x branch.
+
+## Why run translation jobs on master?
+Because existing translations added on master gets removed when you push from other branches. Also, this makes sure you do not accidentally push half-a-year-old strings.
```
Comment (WorldLanguages) [inline content/docs/develop/releasing-patch-update.md:30]: burnnn

**Comment by apple502j** [comment]
Converted to draft; this procedure is too hard to follow in practice

**Comment by apple502j** [comment]
Yeah, not gonna do this

**Comment by TheColaber** [comment]
this will be useful when wl leaves the team

**Comment by WorldLanguages** [comment]
@TheColaber I won't leave the team without writing docs, don't worry :P

**Comment by TheColaber** [comment]
yay 🎉🎉🎉🎉🎉
